lib/auth/access_request_test.go

```diff
@@ -1518,3 +1519,122 @@ func TestUpdateAccessRequestWithAdditionalReviewers(t *testing.T) {
 		})
 	}
 }
+
+func TestAccessRequest_AssumeStartTime(t *testing.T) {
+	modules.SetTestModules(t, &modules.TestModules{TestBuildType: modules.BuildEnterprise})
+	ctx, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
+
+	testPack := newAccessRequestTestPack(ctx, t)
+
+	const requesterUserName = "requester"
+	requester := TestUser(requesterUserName)
+	requesterClient, err := testPack.tlsServer.NewClient(requester)
+	require.NoError(t, err)
+
+	t.Cleanup(func() { require.NoError(t, requesterClient.Close()) })
+
+	clock := clockwork.NewFakeClock()
```

Comment:
Do you need this clock? The clock that your fake server is using is going to be a different clock anyway.